x-pack/plugins/dashboard_enhanced/public/services/drilldowns/actions/drilldown_shared.ts

```diff
@@ -75,3 +75,5 @@ export const createDrilldownTemplatesFromSiblings = (
 
   return templates;
 };
+
+export const drilldownMaxWidth = 500;
```

Comment:
Sorry!! Can you rename this `const` to align with our constant naming convention? And perhaps change its value as shown in https://github.com/elastic/kibana/pull/189009#discussion_r1693611487? (I edited the comment so sorry if you didn't see that originally)

```suggestion
export const DRILLDOWN_MAX_WIDTH = 500;
```

Reply:
Ah sorry yes I missed that - fixed in [b1c4e50](https://github.com/elastic/kibana/pull/189009/commits/b1c4e504759f2c7ee78e57ba3fbb29e62ddf2425)